vars/cosaParallelCmds.groovy

```diff
@@ -0,0 +1,27 @@
+// Run cosa commands in parallel
+// Available parameters:
+//    cosaDir:        string    -- cosa working directory
+//    isBuild:        boolean   -- run build commands using `cosa buildextend-`
+//                                 e.g pass to runCommands only:
+//                                 ['metal', 'metal4k', 'live', 'AWS']
+//    runCommands:  []string    -- list of commands to run
+//    stage:          string    -- CI stage name
+//    user:           string    -- user account to run cosa commands
+
+def call(params = [:]) {
+    def runCommands = params['runCommands'];
+
+    if (params['stage']) {
+        stage(params['stage'])
+    } else {
+        stage("Run")
+    }
```

Comment:
Does this actually wrap the parallel stages or does it just result in an empty stage before the parallel runs? Normally, to define a stage you'd pass the stage code as a close (i.e. `stage("foo") { ... }`).

Maybe let's rework this as:

```groovy
stageName = params.get('stage', "Run")
stage(stageName) {
    parallel ...
}
```

?

Reply:
Actually, I think we should just delete that ability entirely. Let's leave it up to the caller to wrap the call to `cosaParallelCmds` in a stage if they'd like. WDYT?